src/core/catalog.js

```diff
@@ -638,10 +639,39 @@ class Catalog {
       return { name: stringToPDFString(nestedName), order: nestedOrder };
     }
 
+    function parseRBGroups(rbgrps) {
```

Comment:
*Similar comments also applies elsewhere in the patch.*

Shortening this doesn't seem entirely necessary (nor all that readable).
```suggestion
    function parseRBGroups(rbGroups) {
```